requirements.txt

```diff
@@ -1,10 +1,11 @@
-bibtexparser<=1.2.0 # up to date on 07-2021
-psutil<=5.8.0 # Updated to latest on 07-2021
-configparser<=5.0.2 # Updated to latest on 07-2021
-matplotlib==3.0.3; python_version == '3.5'  # Final version
-matplotlib==3.2.2; python_version >= '3.6'  # Latest available 3.4.2 on 07-2021
-numpy<=1.18.4 # Latest available 1.20.3 on 07-2021. Careful. Xmipp is compiled agains numpy!!
-pillow<=8.2.0 # Updated to latest on 07-2021
-requests<=2.25.1 # up to date on 07-2021
-mpi4py <= 3.0.3 # up to date on 07-2021
-tkcolorpicker<=2.1.3 # up to date on 07-2021
\ No newline at end of file
+bibtexparser<=1.2.0 # up to date on 07-2022
+psutil<=5.9.1 # # up to date on 07-2022
+configparser<=5.2.0 # # up to date on 07-2022
+matplotlib==3.3.4; python_version == '3.6'
+matplotlib==3.5.2; python_version >= '3.7' # up to date on 07-2022
+numpy<=1.23.0 # up to date on 07-2022. Careful. Xmipp is compiled agains numpy!
```

Comment:
Shouldn't we recompile xmipp against a new version of numpy?

Reply:
If I install the latest scikit-learn and compile devel xmipp without cuda/opencv, it does compile without errors. This is with gcc 11.3, openmpi 4.1.4. Latest numpy 1.23.0

Reply:
Did you compiled it again the libraries in conda or system wide? Can you share more tips about Xmipp compilation?

Reply:
Hi @delarosatrevin , I only made a new conda env with updated libs. The only problem with xmipp compilation I keep encountering is that it cant detect libhdf5 from conda properly. So I had to export the path to LD_LIBRARY_PATH